### PR TITLE
Confidence Intervals 2

### PR DESCRIPTION
**Pull request by JohnnyDoorn**

### CODE REVIEW

**Comment by JohnnyDoorn** [comment]
@vandenman I made a new PR to deal with the merge conflicts - I think it's good to go now!

File: R/LSTconfidenceIntervals.R [hunk 1]
```diff
@@ -0,0 +1,399 @@
...
+  
+  meanCI <- jaspContainer[["computedConfidenceIntervals"]][["object"]]
+  tableDat <- data.frame(mu = options$mu,
+                         sigma = options$sigma,
+                         Repetitions = options$nReps,
+                         n = options$n,
+                         coverage = sum(meanCI[["successfulCI"]]),
+                         coveragePercentage = mean(meanCI[["successfulCI"]])
+  
+  resultTable$setData(tableDat)
```
Comment (vandenman) [inline R/LSTconfidenceIntervals.R:101]: not sure why the unit tests don't complain about this, but shouldn't there be a `)` here that closes the `data.frame(`? The last one closes `mean(meanCI...`

File: inst/qml/LSTconfidenceIntervals.qml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+			CheckBox
+			{
+				name: "fixAxisTreePlot"
+				label: qsTr("Fix x-axis from ")
+				checked: false
+				childrenOnSameRow: true
+				DoubleField
+				{
+					name: "fixAxisLower"
+					label: qsTr("")
```
Comment (shun2wang) [inline inst/qml/LSTconfidenceIntervals.qml:142]: don't translate empty strings :-)

File: R/LSTconfidenceIntervals.R [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ConfidenceIntervals <- function(jaspResults, dataset = NULL, options) {
```
Comment (vandenman) [inline R/LSTconfidenceIntervals.R:18]: does not match the name in `Description.qml`, that one starts with a a lowercase `c`

File: inst/qml/LSTconfidenceIntervals.qml (hunk 2 above)
Comment (JohnnyDoorn) [reply inst/qml/LSTconfidenceIntervals.qml]: ahh haha so that's what that does. Just removed it!

File: R/LSTconfidenceIntervals.R (hunk 3 above)
Comment (JohnnyDoorn) [reply R/LSTconfidenceIntervals.R]: I now have it as LSTconfidenceIntervals

File: R/LSTconfidenceIntervals.R (hunk 1 above)
Comment (JohnnyDoorn) [reply R/LSTconfidenceIntervals.R]: hahaha yeah very weird (and sloppy..); I just closed the bracket.

**Review by vandenman — CHANGES_REQUESTED** [review]
rest of the code looks fine to me, although I did discover a bug in jaspBase and jaspGraphs that I now have to fix :upside_down_face:

**Review by vandenman — APPROVED** [review]

**Comment by JohnnyDoorn** [comment]
> rest of the code looks fine to me, although I did discover a bug in jaspBase and jaspGraphs that I now have to fix upside_down_face

I love it when my imcompetence uncovers random JASP bugs ^_^

**Pull request merged**